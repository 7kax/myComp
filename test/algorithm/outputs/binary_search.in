2 10 23 33 46 50 69 77 88 99
69
12
2
