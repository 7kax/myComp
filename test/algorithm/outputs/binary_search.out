Found at index: 6
Not found
Found at index: 0
